sound/soc/sof/ops.h

```diff
@@ -470,6 +471,19 @@ snd_sof_set_mach_params(const struct snd_soc_acpi_mach *mach,
 		sof_ops(sdev)->set_mach_params(mach, dev);
 }
 
+static inline void
+snd_sof_register_clients(struct snd_sof_dev *sdev)
+{
+	if (sof_ops(sdev) && sof_ops(sdev)->register_clients)
+		sof_ops(sdev)->register_clients(sdev);
+}
+
+static inline void
+snd_sof_unregister_clients(struct snd_sof_dev *sdev)
+{
+	sof_client_dev_unregister_all(sdev);
```

Comment:
mmh, sorry... This looks really strange. We can remove `sof_client_dev_unregister_all()` and just call it `snd_sof_unregister_clients()`, cannot we? And if you make a new iteration, you might as well add `errno.h` and `module.h` to your headers in `.c` files.